README.md

```diff
@@ -73,6 +73,7 @@
 - [dAppTemplate_MEAN](https://github.com/CityOfZion/dAppTemplate_MEAN) - Interface for communicating with NEO nodes, to help with development of a dApp.
 - [Java smart contracts](https://github.com/neo-project/examples-java) - Examples of smart contracts written in Java from NEO core team.
 - [Kotlin smart contracts](https://github.com/neo-project/examples-kotlin) - Examples of smart contracts written in Kotlin from NEO core team.
+- [Python smart contracts](https://github.com/CityOfZion/awesome-neo/resources/python-examples.md#code-examples) - Examples of smart contracts written in Python, from the Neo Python core team and DApp competition entrants.
```

Comment:
Hi there,

I think this link won't work; between `/awesome-neo/` and `resources/` you should put `blob/master/`, so the final link would be: 
`https://github.com/CityOfZion/awesome-neo/blob/master/resources/python-examples.md#code-examples`
Could be wrong though but this is my experience at least :)

Reply:
`DApp` should be `dApp` to keep the file consistent